Update German translation

daPhipz opened the pull request:
I updated the German translation and made the string for `translator_credits` translatable. This is commonly used for GNOME applications, as it enables showing only the people who were involved in the translation the user sees on their system.

As I (obviously) use German on my system, I would like to see how this impacts an English build of the Extension - especially because English is the default for all locales that don't have a translatiion yet. Does it say "Translation by translator-credits" in the about dialogue? If so, I'll ask on the GNOME localization IRC about how to deal with that.

stuarthayhurst commented:
Thanks :)

daPhipz commented:
@stuarthayhurst How does your credits screen look like now?

stuarthayhurst commented:
I tested your branch before I merged, it worked

![image](https://user-images.githubusercontent.com/33733427/120382339-6cb0ee00-c31b-11eb-8786-c8e501d2995d.png)

daPhipz commented:
Weird - why does it show my name? It should show something else, since you are not using the German translation.
I also added the year in my commit, so maybe this is an old version? Make sure to reload the shell - what does it say then?

stuarthayhurst commented:
Yep, I've reloaded the shell and tried several times

stuarthayhurst commented:
What's it supposed to say instead?

daPhipz commented:
I changed the line in the [source code](https://github.com/stuarthayhurst/alphabetical-grid-extension/pull/13/commits/66ea6b098be5d3e6924f433521d3a67236591335#diff-cbd3ad962010bb97aaa7d45e4a08400e6ee51212f9c9b3803aa7d6b513fe90b7R35) to

```
translator_credits: _('translator-credits')
```

The string `translator-credits`, (the one in brackets) is now translatable, which enables translators of different languages to credit themselves.
If we had a French and a German translation, I (as a German) would only see me (the German translator) in the credits, and a French user would only see the French translator in the credits, since I am not mentioned in `po/fr.po`.

My question now is  - what do you see as a person who uses the English locale on their system? There is no obviously no translation for English, because the source code already is English. I suspect therefore that you should see something like

```
Translated by translator-credits
```
since this is the string we used in the source code, but you see something different.
Your screenshot made me wonder too, because I changed the German translation to

```
#: prefs.js:35
msgid "translator-credits"
msgstr "Philipp Kiemle <philipp.kiemle@gmail.com>, 2021"
```

which looks like this in the UI:

![Bildschirmfoto vom 2021-06-02 08-46-38](https://user-images.githubusercontent.com/30795174/120436318-14133c80-c37f-11eb-87eb-405f6fc1afe7.png)

That's why I am so perplexed ;) Do you understand my problem now?

daPhipz commented:
Ah yes, thank you very much for checking!
